supertokens_python/recipe/emailpassword/recipe.py

```diff
@@ -49,26 +55,37 @@
 class EmailPasswordRecipe(RecipeModule):
     recipe_id = 'emailpassword'
     __instance = None
+    email_delivery_ingredient: EmailDeliveryIngredient[TypeEmailPasswordEmailDeliveryInput]
 
     def __init__(self, recipe_id: str, app_info: AppInfo,
                  sign_up_feature: Union[InputSignUpFeature, None] = None,
                  reset_password_using_token_feature: Union[InputResetPasswordUsingTokenFeature, None] = None,
                  email_verification_feature: Union[InputEmailVerificationConfig, None] = None,
                  override: Union[InputOverrideConfig, None] = None,
-                 email_verification_recipe: Union[EmailVerificationRecipe, None] = None):
+                 email_verification_recipe: Union[EmailVerificationRecipe, None] = None,
+                 email_delivery: Union[EmailDeliveryConfig[TypeEmailPasswordEmailDeliveryInput], None] = None,
+                 email_delivery_ingredient: Union[EmailDeliveryIngredient[TypeEmailPasswordEmailDeliveryInput], None] = None):
         super().__init__(recipe_id, app_info)
         self.config = validate_and_normalise_user_input(self, app_info, sign_up_feature,
                                                         reset_password_using_token_feature,
-                                                        email_verification_feature, override)
-        if email_verification_recipe is not None:
-            self.email_verification_recipe = email_verification_recipe
-        else:
-            self.email_verification_recipe = EmailVerificationRecipe(recipe_id, app_info,
-                                                                     self.config.email_verification_feature)
+                                                        email_verification_feature, override, email_delivery)
         recipe_implementation = RecipeImplementation(
             Querier.get_instance(recipe_id))
         self.recipe_implementation = recipe_implementation if self.config.override.functions is None else \
             self.config.override.functions(recipe_implementation)
+
+        if email_delivery_ingredient is None:
+            self.email_delivery_ingredient = EmailDeliveryIngredient(self.config.email_delivery(self))
+        else:
+            self.email_delivery_ingredient = email_delivery_ingredient
+
+        if email_verification_recipe is not None:
+            self.email_verification_recipe = email_verification_recipe
+        else:
+            self.email_verification_recipe = EmailVerificationRecipe(recipe_id, app_info,
+                                                                     self.config.email_verification_feature,
+                                                                     email_delivery_ingredient=self.email_delivery_ingredient)
+
```

Comment:
"TypeEmailPasswordPasswordResetEmailDeliveryInput" is incompatible with "TypeEmailVerificationEmailDeliveryInput"